common/src/main/resources/tachyon-default.properties

```diff
@@ -70,7 +70,7 @@ tachyon.worker.memory.size=128MB
 tachyon.worker.network.netty.watermark.high=32KB
 tachyon.worker.network.netty.watermark.low=8KB
 tachyon.worker.allocate.strategy=MAX_FREE
-tachyon.worker.evict.strategy=LRU
+tachyon.worker.evict.strategy=tachyon.worker.block.evictor.LRUEvictor
```

Comment:
Could you rename this property as "tachyon.worker.evict.strategy.class"? (basically, add the suffix ".class")

Could you also rename the Constant to match the property name?